dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/BarGraphUtility/BarBrushHelper.cs

```diff
@@ -30,111 +30,91 @@
 namespace BarGraphUtility
 {
     // Create brushes to fill the bars. 
-    public class BarBrushHelper
+    sealed class BarBrushHelper
     {
-        private Compositor compositor;
-        private Random rand;
+        private readonly Compositor _compositor;
+        private readonly Random _rand;
 
         public BarBrushHelper(Compositor c)
         {
-            compositor = c;
-            rand = new Random();
+            _compositor = c;
+            _rand = new Random();
```

Comment:
Any readonly fields that you can initialize inline should be initialized inline, not in the constructor.

Reply:
updated